openapi-generator.yaml

```diff
@@ -0,0 +1,16 @@
+inputSpec: openapi.json
+output: ./
+
+generatorName: go
+
+gitUserId: sijoma
+gitRepoId: console-customer-api-go
+
+validateSpec: false
+
+modelNameMappings:
+  # `ClusterStatus` conflicts with the autogenerated Cluster's `status`
+  ClusterStatus: ClusterComponentStatus
```

Comment:
Nice.  Have not found this.

Reply:
It's a bit of a spoiler for the actual update, but yeah, this will help...